app/helpers/reference_helper.rb

```diff
@@ -0,0 +1,38 @@
+module ReferenceHelper
+
+  def after_desired_date?
+    Time.zone.today > Date.parse('2015-12-31')
```

Comment:
@novotnyjakub 

> Can we put this to settings.yml and read it from the Settings class? Maybe the settings.yml should even read the value from ENV so that we don't have to do a full release if this changes.

We know where this one value resides, in this helper. The amount of time and seeing how we already have a ticket to delete this feature after 1st January: https://www.pivotaltracker.com/story/show/109815848 I'd argue that this would not be worth it, even though from the design perspective your suggestion is cleaner, but only if this was going to be a feature that would stay. Seeing as it's only temporary, I'd say to leave it as is.